datastreamer/streamserver.go

```diff
@@ -727,7 +733,6 @@ func (s *StreamServer) broadcastAtomicOp() {
 				}
 			}
 		}
-		s.mutexClients.RUnlock()
 
```

Comment:
We need to guard [this](https://github.com/0xPolygon/zkevm-data-streamer/blob/46a2c2bda0773344c48aad42ddfb794cb2abe21e/datastreamer/streamserver.go#L750-L751) line by the lock as well (we are accessing the `len(s.clients)`), wdyt?